to_production.txt

```diff
@@ -1,3 +1,17 @@
-# Upgrade
+# New
+ofl/sixtyfourconvergence # https://github.com/google/fonts/pull/7919
```

Comment:
Let's remove this COLRv1 family from the list.  We need time to prevent this from landing in Workspace editors

Reply:
Done!